package.json

```diff
@@ -9,6 +9,7 @@
     "chart.js": "2.9.4",
```

Comment:
A chart.js libeket akkor nem kéne törölni?

Reply:
Nem én akarom eldönteni, hogy ki legyen-e véve a JavaScript, ez egy alternatíva, illetve jelenleg mind a kettő üzemel: JavaScript engedélyezése esetén az előző, anélkül az új..

Míg amúgy nem feature complete a lib, merte axis labelt nem tud, pedig azt még én is megírtam kézzel a PoC-ban, de roadmapen van (és volna rá hack de azt nem ajánlom).